itou/metabase/management/commands/sql/019_candidatures_echelle_locale.sql

```diff
@@ -1,18 +1,24 @@
-    
-/* 
-     
+/*
+
 L'objectif est de créer une table qui contient des informations à l'échelle locale (bassin d'emploi, epci, etc)
-    
+
 */
-    
-with candidatures_p as ( 
-    select  
+
+with candidatures_p as (
+    select
         *
     from
-        candidatures 
+        candidatures
+),
+org_prescripteur as ( /* On récupère l'id et le dept des organismes prescripteurs */
```

Comment:
ce commentaire explique ce que font les lignes de code (pourquoi pas, mais c'est assez évident) ; ce serait encore mieux de dire __pourquoi__ on veut cette info :)

Reply:
Je note aussi, car j'avais tendance à juste expliquer sans dire le pourquoi 👍

Reply:
https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/